content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -56,18 +56,21 @@ A cluster administrator creates a number of PVs. They carry the details of the r
 #### Dynamic
 When none of the static PVs the administrator created matches a user's `PersistentVolumeClaim`,
 the cluster may try to dynamically provision a volume specially for the PVC.
-This provisioning is based on `StorageClasses`: the PVC must request a
-[storage class](/docs/concepts/storage/storage-classes/) and
-the administrator must have created and configured that class in order for dynamic
-provisioning to occur. Claims that request the class `""` effectively disable
-dynamic provisioning for themselves.
-
-To enable dynamic storage provisioning based on storage class, the cluster administrator
-needs to enable the `DefaultStorageClass` [admission controller](/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass)
-on the API server. This can be done, for example, by ensuring that `DefaultStorageClass` is
-among the comma-delimited, ordered list of values for the `--enable-admission-plugins` flag of
-the API server component. For more information on API server command line flags,
-please check [kube-apiserver](/docs/admin/kube-apiserver/) documentation.
+This provisioning uses the concept of [storage classes](/docs/concepts/storage/storage-classes/) which requires the
```

Comment:
How about something like this?

"Dynamic provisioning requires a `StorageClass` to be created and configured by an administrator.  PVCs can request a `StorageClass` by specifying its name in the `StorageClassName` field. A nil `StorageClassName` will use the default StorageClass configured in the cluster.  An empty string `StorageClassName` disables dynamic provisioning"